domain/appeal.go

```diff
@@ -126,7 +126,7 @@ type ApprovalAction struct {
 
 type ListAppealsFilter struct {
 	CreatedBy                 string    `mapstructure:"created_by" validate:"omitempty,required"`
-	AccountID                 string    `mapstructure:"account_id" validate:"omitempty,required"`
+	AccountIDs                []string  `mapstructure:"account_ids" validate:"omitempty,required"`
```

Comment:
this will be a breaking change, we can still keep `AccountID` with a "deprecated" comment and add the new `AccountIDs` to keep it backward compatible. Therefore, no implementation changes should be done in ListAppeals API apart from `AccountIDs` filter addition